.env.example

```diff
@@ -16,7 +16,7 @@ SOLIS_CLOUD_API_PORT = '13333'
 GIVENERGY_API_KEY = 'user_givenergy_api_key'
 
 # User needs to add their GivEnergy API details
-SOLARMAN_API_URL = 'https://api.solarmanpv.com/device/v1.0'
+SOLARMAN_API_URL = 'https://home.solarmanpv.com/maintain-s/history/power'
```

Comment:
Did you find any documentation on this URL in the end? It would be good to add as a TODO to switch to the actual API URL itself at some point if there is a benefit to doing so

Reply:
> Did you find any documentation on this URL in the end? It would be good to add as a TODO to switch to the actual API URL itself at some point if there is a benefit to doing so

No there was nothing on this url , it was the prev api.domain whatever url in the official site